stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -442,4 +442,21 @@ class GaussianDropout(Layer[tf.Tensor, tf.Tensor]):
         name: str | None = None,
     ) -> None: ...
 
+class MaxPooling2D(Layer[tf.Tensor, tf.Tensor]):
+    def __init__(
+        self,
+        pool_size: int | tuple[int, int] = (2, 2),
+        strides: int | tuple[int, int] | None = None,
+        padding: Literal["valid", "same"] = "valid",
+        data_format: None | Literal["channels_last", "channels_first"] = None,
+        name: str | None = None,
```

Comment:
The implementation seems to accept `**kwargs` though I'm not sure the purpose

https://github.com/keras-team/keras/blob/v3.3.3/keras/src/layers/pooling/max_pooling2d.py#L96